code/modules/mob/living/carbon/carbon_defines.dm

```diff
@@ -33,6 +33,7 @@
 	var/obj/item/clothing/neck/wear_neck = null
 	var/obj/item/tank/internal = null
 	var/obj/item/clothing/head = null
+	var/obj/item/clothing/wear_suit = null
```

Comment:
Seems very sus to move this to `/carbon` entirely instead of just casting to `/human` when you're checking for it, or instead of using `get_item_by_slot()` to get it

Reply:
It's to allow the anomaly to also work on monkeys, is there any other way to do it?

Reply:
Like I said, either:
- cast to human
  - `if(ishuman(affected mob)); var/mob/living/carbon/human/affected_human = affected mob; affected_human.wear_suit`, etc
- use `get_item_by_slot(ITEM_SLOT_OCLOTHING)`, as it's a mob level proc